<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWidgetSprint</class>
 <widget class="QWidget" name="CWidgetSprint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>1115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(17, 149, 189);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Sprint Info</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_projectName">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>项目名称</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_sprintNo">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>迭代1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAddBacklog">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:1px solid black;
border-radius:4px;
padding:4px;
}
QPushButton::hover{
border:1px solid black;
border-radius:4px;
background-color:lightgray;
padding:4px
}
QPushButton::pressed{
border:1px solid black;
border-radius:4px;
background-color:gray;
padding:4px
}</string>
       </property>
       <property name="text">
        <string>add Backlog</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBurndown">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:1px solid black;
border-radius:4px;
padding:4px;
}
QPushButton::hover{
border:1px solid black;
border-radius:4px;
background-color:lightgray;
padding:4px
}
QPushButton::pressed{
border:1px solid black;
border-radius:4px;
background-color:gray;
padding:4px
}</string>
       </property>
       <property name="text">
        <string>burndown</string>
       </property>
       <property name="icon">
        <iconset resource="ks24_03.qrc">
         <normaloff>:/images/burndown.png</normaloff>:/images/burndown.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFinish">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:1px solid black;
border-radius:4px;
padding:4px;
}
QPushButton::hover{
border:1px solid black;
border-radius:4px;
background-color:lightgray;
padding:4px
}
QPushButton::pressed{
border:1px solid black;
border-radius:4px;
background-color:gray;
padding:4px
}</string>
       </property>
       <property name="text">
        <string>all backlogs are done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGetback">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:1px solid black;
border-radius:4px;
padding:4px;
}
QPushButton::hover{
border:1px solid black;
border-radius:4px;
background-color:lightgray;
padding:4px
}
QPushButton::pressed{
border:1px solid black;
border-radius:4px;
background-color:gray;
padding:4px
}</string>
       </property>
       <property name="text">
        <string>return</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>900</width>
        <height>941</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>5</number>
          </property>
          <item row="0" column="1">
           <widget class="QFrame" name="frame_7">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(182, 182, 182);</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_doing">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(205, 205, 205);
border-radius:5px;
border:none</string>
               </property>
               <property name="text">
                <string>doing</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QFrame" name="frame_10">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(182, 182, 182);
border-top-right-radius:5px;
border-bottom-right-radius:5px;</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_10">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <property name="spacing">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_finished">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(205, 205, 205);
border-radius:5px;
border:none</string>
               </property>
               <property name="text">
                <string>done</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QFrame" name="frame_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(182, 182, 182);
border-top-left-radius:5px;
border-bottom-left-radius:5px;</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_toDo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(205, 205, 205);
border-radius:5px;
border:none</string>
               </property>
               <property name="text">
                <string>backlog</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>917</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ks24_03.qrc"/>
 </resources>
 <connections/>
</ui>
